### PR TITLE
Php ini multiple keys are now allowed

### PR DESCRIPTION
**Pull request by Scriptkiddi**
Added namespacing to variables where needed and added the possibility to have php ini values with the same key since this is needed to enable multiple extensions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ This role needs systemd and an apt based package manager.
 ## Role Variables
 
 To see all vars possible for php-fpm see 
-[php-fpm config page](https://secure.php.net/manual/en/install.fpm.configuration.php)
+[php-fpm config page](https://secure.php.net/manual/en/install.fpm.configuration.php). To use them prefix them with php_fpm_ (but not the pool variables)
```
Comment (SlothOfAnarchy) [inline README.md]: block quotes

File: defaults/main.yml [hunk 2]
```diff
@@ -1,31 +1,30 @@
...
+php_fpm_ping_response: pong
+php_fpm_request_terminate_timeout: 0
+php_fpm_request_slowlog_timeout: 0
+php_fpm_slowlog: log/php-fpm.log.slow
+php_fpm_catch_workers_output: no
+php_fpm_clear_env: Yes
+php_fpm_security_limit_extensions: .php .phar
+php_fpm_access_format: "%R - %u %t \"%m %r\" %s"
+php_fpm_php_ini_values:
+  expose_php: 0
```
Comment (SlothOfAnarchy) [inline defaults/main.yml:30]: This used to be a list, is this intended?

**Review by SlothOfAnarchy — DISMISSED** [review]

File: defaults/main.yml (hunk 2 above)
Comment (Scriptkiddi) [reply defaults/main.yml:30]: it was where do you take that from?

**Comment by Scriptkiddi** [comment]
@SlothOfAnarchy pls rereview

File: defaults/main.yml (hunk 2 above)
Comment (SlothOfAnarchy) [reply defaults/main.yml:30]: 'Changed Files' shows the difference.

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**